lib/pages/new_private_chat/search_contacts_controller.dart

```diff
@@ -7,6 +7,7 @@ import 'package:fluffychat/pages/search/get_contacts_controller.dart';
 import 'package:fluffychat/presentation/converters/presentation_contact_converter.dart';
 import 'package:flutter/material.dart';
 import 'package:matrix/matrix.dart';
+import 'package:pull_to_refresh/pull_to_refresh.dart';
 
 mixin class SearchContactsController {
```

Comment:
should have name end with mixin

Reply:
it can still be used as a standalone controller

Reply:
`SearchContactsMixinController`

Reply:
done